apps/example-app-vue/project.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
The official Vite Vue app doesn't have a `project.json` so I create a new one with the configs from the Nx official docs --> https://nx.dev/packages/vite/documents/set-up-vite-manually

Reply:
good solution!